mixer/adapter/stdio/stdio.go

```diff
@@ -62,11 +64,11 @@ func (h *handler) HandleLogEntry(_ context.Context, instances []*logentry.Instan
 
 		for _, varName := range h.logEntryVars[instance.Name] {
 			if value, ok := instance.Variables[varName]; ok {
-				fields = append(fields, field(varName, value))
+				fields = append(fields, zap.Any(varName, value))
```

Comment:
curious: have the IP_ADDRESS issues been resolved?

Reply:
Yep, we pass the right thing to adapters now. Verified by my tests.